lpcompiler.pas

```diff
@@ -2264,6 +2273,9 @@ function TLapeCompiler.ParseVarBlock(OneOnly: Boolean = False; ValidEnd: EParser
           VarDecl.VarDecl := addLocalVar(VarType, Identifiers[i]);
 
         VarDecl.Default := nil;
+        if (not (lcoHints in FOptions)) then
+          VarDecl.VarDecl.Used := duIgnore;
```

Comment:
I think this is better than adding a parameter to `StackInfo.addVar`. But probably better to use `Ignore` by default and then switch to `False` to enable hints.